### PR TITLE
postMeasurements with access token, fixes #378

### LINKED ISSUE #378: Authentication Error when Uploading measurements from csv/json file to a secured box
### Expected Behavior
Measurements should be uploaded from csv/json file, authentication already happened when logging in. 

### Actual Behavior
When uploading measurements manually via the UI (Dashboard -> Data Upload) to a senseBox where authentication is activated, an authentication error occurs. This is likely because the function was not updated after authentication for posting measurements was implemented. 

### Environment
- OS: 
- Browser & Version: Firefox

### PR DESCRIPTION
**Pull request by Thiemann96**
### Fix or Enhancement and description?
When posting measurements from OSM (e.g. via manual data upload) an access token is provided. 

### Environment
- OS: Write here
- Browser & Version: Write here

### CODE REVIEW

File: app/scripts/controllers/account.box.dataupload.js [hunk 1]
```diff
@@ -57,9 +57,8 @@
     function submitData () {
       vm.success = false;
       vm.error = '';
-
       OpenSenseMapAPI
-        .postMeasurements($state.params.id, vm.measurementData, vm.dataFormat)
+        .postMeasurements($state.params.id, vm.measurementData, vm.dataFormat,$state.params.box.access_token)
```
Comment (ubergesundheit) [inline app/scripts/controllers/account.box.dataupload.js:61]: lets keep it readable :)

```suggestion
        .postMeasurements($state.params.id, vm.measurementData, vm.dataFormat, $state.params.box.access_token)
```

File: app/scripts/services/opensensemapapi.js [hunk 2]
```diff
@@ -92,12 +92,12 @@
         .catch(failed);
     }
 
-    function postMeasurements (boxId, measurements, format) {
+    function postMeasurements (boxId, measurements, format,access_token) {
```
Comment (ubergesundheit) [inline app/scripts/services/opensensemapapi.js:95]:
```suggestion
    function postMeasurements (boxId, measurements, format, access_token) {
```

File: app/scripts/services/opensensemapapi.js [hunk 3]
```diff
@@ -92,12 +92,12 @@
...
       var url = getUrl() + '/boxes/' + boxId + '/data';
 
       return $http
         .post(url, measurements, {
-          headers: { 'content-type': format }
+          headers: { 'content-type': format ,'Authorization': access_token}
```
Comment (ubergesundheit) [inline app/scripts/services/opensensemapapi.js:100]: Have you tested this for boxes which do not require authorization?

```suggestion
          headers: { 'content-type': format , 'Authorization': access_token}
```

**Review by ubergesundheit — CHANGES_REQUESTED** [review]

File: app/scripts/services/opensensemapapi.js (hunk 3 above)
Comment (Thiemann96) [reply app/scripts/services/opensensemapapi.js:100]: I did ! Everything was working as expected

**Comment by Thiemann96** [comment]
I have tested it with boxes where auth is activated and deactivated also. Everything is working as expected

File: app/scripts/services/opensensemapapi.js [hunk 4]
```diff
@@ -92,12 +92,12 @@
...
     }
 
-    function postMeasurements (boxId, measurements, format) {
+    function postMeasurements (boxId, measurements, format, access_token) {
       var url = getUrl() + '/boxes/' + boxId + '/data';
 
       return $http
         .post(url, measurements, {
-          headers: { 'content-type': format }
+          headers: { 'content-type': format , 'Authorization': access_token}
```
Comment (ubergesundheit) [inline app/scripts/services/opensensemapapi.js:100]: Sorry, one last nitpick

```suggestion
          headers: { 'content-type': format , 'Authorization': access_token }
```

Comment (Thiemann96) [reply app/scripts/services/opensensemapapi.js:100]: No worries, "Ordnung muss sein"

**Review by ubergesundheit — APPROVED** [review]
Thanks :)

**Pull request merged**